cmd/tailscale/cli/version.go

```diff
@@ -23,14 +23,16 @@ var versionCmd = &ffcli.Command{
 		fs := newFlagSet("version")
 		fs.BoolVar(&versionArgs.daemon, "daemon", false, "also print local node's daemon version")
 		fs.BoolVar(&versionArgs.json, "json", false, "output in JSON format")
+		fs.BoolVar(&versionArgs.withLatest, "with-latest", false, "include latest released version output")
```

Comment:
say something about how it hits the network?

also, the "with" here doesn't seem consistent: "daemon" isn't "with-daemon". Not that I have other suggestions at the moment.

Reply:
Changed flag to `--upstream` and updated comment to say it's fetching the version from `pkgs.tailscale.com`